ui/models/runnableJob.js

```diff
@@ -24,23 +23,26 @@ export default class RunnableJobModel {
       return rawJobs;
     }
 
-    Object.entries(rawJobs).forEach(([key, value]) => {
-      jobList.push({
-        build_platform: value.platform || '',
-        build_system_type: 'taskcluster',
-        job_group_name: value.groupName || '',
-        job_group_symbol: value.groupSymbol || '',
-        job_type_name: key,
-        job_type_symbol: value.symbol,
-        platform: value.platform || '',
-        platform_option: Object.keys(value.collection).join(' '),
-        ref_data_name: key,
-        state: 'runnable',
-        result: 'runnable',
-        push_id: params.push_id,
-        id: escapeId(params.push_id + key),
-      });
-    });
-    return jobList;
+    return Object.entries(rawJobs).reduce(
```

Comment:
Would `.map([key, value] => new JobModel(...))` be cleaner here instead of the reduce?

Reply:
Heh, you're totally right.  Thanks for catching that.  :)